recipe/meta.yaml

```
@@ -1,35 +1,45 @@
 {% set name = "solvation_analysis" %}
-{% set version = "0.1.5" %}
+{% set version = "0.3.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/solvation_analysis-{{ version }}.tar.gz
-  sha256: f33b867f42aace0d4f9199da86b21d2bd04a3cf9dab8ba039e0ee21bdd81b919
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/solvation-analysis-{{ version }}.tar.gz
+  sha256: 8dbf81b973e5e02c8dcd8e9247a794dd50bda701f4ce05fa681cd67e0bdb0ef8
 
 build:
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
   number: 0
 
 requirements:
+  build:
+    - versioneer-518
   host:
-    - python >=3.7
+    - python >=3.8
     - pip
+    - pathlib
+    - pytest
+    - rdkit
+    - versioneer-518
   run:
-    - python >=3.7
-    - numpy >=1.16.0
+    - python >=3.8
+    - numpy >=1.18.0
```

```suggestion
    - numpy >=1.20.0
```
This doesn't match your pyproject.toml